### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by OMMediaGroupFileRepos** [t1m0thyj/WinDynamicDesktop#477] [Ideas]
Hello, i made an Discord server for WinDynamicDesktop for support, could this be added to the website and page?

So people can get help easier.

**Comment by t1m0thyj** [comment]
Hello, thanks for your interest in a Discord server. I created one for WDD several months ago and linked it here on GitHub Discussions but only 1 person joined so there didn't seem to be enough interest.

I don't plan for there to be an official Discord server at this time. You can get help through GitHub issues or the WDD subreddit.

**Reply by OMMediaGroupFileRepos** [reply]
Could i hold the current server?

i could spread WDD to my friends and build an unofficial community.
so i could maintain the Discord server with people that have interest.

and let them get help etc.

**Reply by ghost** [reply]
yes please make discord server, we can help each other out much more efficient.

**Reply by OMMediaGroupFileRepos** [reply]
Add me: OfficieelMika#7371
i will make the server ready.
